added documentation for creating a reply note via extrenal api (#126)
affinity/slate: source/includes/_notes.md

account that owns the Gmail mesage.) A note associated with an email cannot have any
`content`.

> It is possible to create a <strong>reply note</strong> for an existing note by sending `parent_id` as one of the parametrs. The parent note should exist and shuold not be a reply note, having a not null parent_id, itself. It is possible for a note to have multiple replies. In the presence of the parent_id, opportunity_ids, person_ids and organization_ids will be ignored.

Reviewer: ```suggestion
It is possible to create a <strong>reply note</strong> for an existing note by sending `parent_id` as one of the parameters. The parent note should exist and should not be a reply note, having a not null `parent_id` itself. It is possible for a note to have multiple replies. `opportunity_ids`, `person_ids`, and `organization_ids` will be ignored when providing a `parent_id`.
```

Some typo and formatting fixes. Also reworded that last sentence to make it clearer which fields are being ignored; this could be misconstrued if read with the wrong intonation.

Author: I just installed it, and updated my VS code setting page. Thank you very much VSCode bro  ;)
